src/library/Hooks/useTimeLeft/utils.ts

```diff
@@ -28,15 +29,19 @@ export const getDuration = (toDate: Date | null): TimeleftDuration => {
   });
 
   const months = d?.months || 0;
-  const days = d?.days || 0;
+  let days = d?.days || 0;
   const hours = d?.hours || 0;
   const minutes = d?.minutes || 0;
   const seconds = d?.seconds || 0;
   const lastHour = months === 0 && days === 0 && hours === 0;
   const lastMinute = lastHour && minutes === 0;
 
+  if (months > 0) {
+    const _days = differenceInDays(toDate, Date.now());
+    days += _days;
```

Comment:
This is adding days of the duration twice. I think this should be overwriting `days` instead of adding onto it.